azure/src/test/java/sunstone/azure/armTemplates/di/AzVmTests.java

```diff
@@ -11,40 +11,37 @@
 import sunstone.azure.armTemplates.AzureTestConstants;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static sunstone.azure.armTemplates.di.AzVmTests.group;
 
 
 @WithAzureArmTemplate(parameters = {
         @Parameter(k = "virtualMachineName", v = AzureTestConstants.instanceName),
         @Parameter(k = "imageRefId", v = AzureTestConstants.IMAGE_REF)
 },
-        template = "sunstone/azure/armTemplates/eap.json", group = group, perSuite = true)
```

Comment:
Can we have unique test class specific group? Like in `MultipleAzArmTemplatesRepetableTest`.